TrustModel.md

```diff
@@ -0,0 +1,113 @@
+# Trust Model for Confidential Containers
+A clear definition of trust for the confidential containers project is needed to ensure the 
+components and architecture deliver the security principles expected for cloud native 
+confidential computing. It provides the solid foundations and unifying security principles 
+against which we can assess architecture and implementation ideas and discussions.
+
+ Trust modeling can be defined[^5] as :
+
+    A trust model identifies the specific mechanisms that are necessary to respond to a specific 
+    threat profile.
+
+    A trust model must include implicit or explicit validation of an entity's identity or the 
+    characteristics necessary for a particular event or transaction to occur.
+
+ The trust model also helps determine the location and direction of the trust boundaries where a 
+[trust boundary](https://en.wikipedia.org/wiki/Trust_boundary) describes a location where 
+ program data or execution changes its level of "trust", or where two principals with different 
+ capabilities exchange data or commands. Specific to Confidential Containers is the trust 
+ boundary that corresponds to the boundary of the Trusted Execution Environment (TEE). Generally, 
+ the TEE side of the trust boundary will be hardened to prevent the violation of the trust 
+ boundary.
+
+In order to describe and understand particular threats we need to establish trust boundaries and 
+trust models relating to the key aspects, components and actors involved in Cloud Native 
+Confidential Computing. We explore trust using different orthogonal ways of considering cloud 
+native approaches when they use an underlying Trusted Execution Environment technology and 
+identifying where there may be considerations to preserve the value of using a TEE.
+
+<!--
+   Follow on pull requests will list here subsequent sections corresponding to orthogonal ways 
+of considering cloud native approaches
+Specifically including
+   - Personas
+   - Lifecycle
+   - Trusted Execution Boundary Location
+   - Secured v unsecured TEE Base Image
+-->
+
+
+Further documentation will highlight specific threat vectors in detail, considering risk, 
+impact, mitigation etc as the project progresses. The Security Assurance section, Page 31, of 
+CNCF Cloud Native Security Whitepaper[^2] will guide this more detailed threat vector effort.
+
+
+### Related Prior Effort
+
+Confidential Containers brings confidential computing into a cloud native context and should 
+therefore refer to and build on trust and security models already defined.
+
+For example: 
+
+- Confidential Computing Consortium (CCC) published 
+  "[A Technical Analysis of Confidential Computing](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/CCC-Tech-Analysis-Confidential-Computing-V1.pdf) [^1]" 
```

Comment:
Yes bit of duplication, I need to choose one or the other!.
Open to whatever people feel, The footnotes probably made more sense when it was one long document and saved repeating a link but with the model heading towards separate documents for aspects of it, maybe it makes less sense to use footnotes.
What do people think , just links or footnotes?

Reply:
I don't think footnotes are needed when links are present.

Reply:
If a single link without other comment info is enough, just use it, even it will be repeated. If we need to refer to a specific page number or section name of material, footnote with page number or section name will be useful, or use this form:`The Security Assurance section, Page 31, of [CNCF Cloud Native Security Whitepaper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/CNCF_cloud-native-security-whitepaper-Nov2020.pdf)`

Reply:
Removed the footnotes tidied up references